test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -621,29 +622,50 @@ def validate(self, model: torch.fx.GraphModule) -> None:
             def get_supported_operators(cls) -> List[OperatorConfig]:
                 pass
 
-        m = TestHelperModules.ConvMaxPool2d()
-        x = torch.rand(1, 2, 14, 14)
-        example_inputs = (x,)
-        node_occurrence = {
-            # two for input of maxpool
-            # one for input for maxpool
-            # one for output of maxpool
-            torch.ops.quantized_decomposed.quantize_per_tensor: 4,
-            torch.ops.quantized_decomposed.dequantize_per_tensor: 4,
-        }
-        node_list = [
-            torch.ops.quantized_decomposed.dequantize_per_tensor,
-            torch.ops.quantized_decomposed.dequantize_per_tensor,
-            torch.ops.aten.convolution.default,
-            torch.ops.quantized_decomposed.quantize_per_tensor,
-            torch.ops.quantized_decomposed.dequantize_per_tensor,
-            torch.ops.aten.max_pool2d_with_indices.default,
-            torch.ops.quantized_decomposed.quantize_per_tensor,
-            torch.ops.quantized_decomposed.dequantize_per_tensor,
-        ]
-        self._test_quantizer(
-            m, example_inputs, BackendAQuantizer(), node_occurrence, node_list
-        )
+        for use_maxpool2d in [True, False]:
```

Comment:
maybe just split this to two tests?

Reply:
Thanks for the suggestion. Split into two tests of `test_wo_annotate_conv_output_quantizer` and `test_max_pool2d_quantizer`. Please kindly help to take a look again @jerryzh168.